mappings/337f99bf2d3f6ed51793e0bbbcc41c1809fe34b88458c6c16525f78c43424f59.json

```diff
@@ -0,0 +1,64 @@
AXqnMOCsRXTyaTESVF8JCAHEohk4GKY3MJSDHtAELwxDBaDwQBHJugRGGcw5CbLhmA5cDAyPkICwJ3ZPICxI/fhCYwLMzz63AIIYfMskAgEmNyn6aZaZwAYHo6vjwO/uXxMaV5Gr48PbKUBQBKYppn+OIpC56ynyHFl2Wc5un81999uAKZA7ylvJrprtYCBaBQIEXPmbQ/4wtAqixumVdcFgq7hQL4EylA6ThHBJgh7HPIK3bnQMXVDBlIQISrbR4FjS6KAAIYAMrzA888wOTDYhU4onJH+HoYvhBzAhXyIxleURk4HI4sAyFkYCEQYyRAzstADtEyAxN+70cY8cAwzEQW7n1iIjPDAJ55nmDweJ/OpZAA0vICzPCc8hUxk3lLQQOCAg5PKaVQPJ4bxSuypcy1lHIpAFBhcLtdpZCgkDLQBVIilUyGvNr9UGvAwTmvLGktEHB7H2MIe+8iIrIV+4Cy0iEOQYAImgEYCSkDSlhtEBY+HUO4Y3nNAAOvJCAeDodwFwhglxcW8JHCyHmAeYosPJLmyR8Zh1CW8zQNeGCGjwd/PzADZfrIspTyiek8Ls9/NBgg90BmowySEcjCggfg5rfKyu6JE6WczwUK5/KN6lKplUKhlETOQFBqSgCFENi2swNXgQCQiMl7wui+8eT+EI2HIIDgWLsuORwXAGIgDzCGCAuRxPD7MXKlcRxDQGEAMoQBgEAYIAEwHQ7xAMkmXtJMOeE5zwCMA8xEYPAzcSQxjwALE5zjQIQZAAbwLzMjw8szQwIC5JhXHDe/B7QOACklUkrHY7qkAiSAnH4jyhW5cirlXC6XcykUMpCzAErSGBM5Z/AhwMrBONZYeIqFCAFCCDHuo/hO3CYjmQ5QEsADADm3AHjWaH0cfgxhEPEDDMMIKOHwdwdyzuEuhBA2FzkfC5CImP9yWIb7fwT49ASkZfiJp5f7nz/96fDT/K+/f+R/P7+//3jml2X48vufYfo/XogpPn5k5mZw/jzN8MfP//HPPWRMYnXHWZ3uDEB1i6uceQqEzC2YFCG9QT3CMcQERHyBlNm4PCUgppJzCPibxqkOzsBGolfd7WMMB49xjrhDENzacNf+9Pzz8ZnfzZ94+eunYeHuMJa/fonT45/5L+6nJ/7yZ7Dt7sf7xJenhSEcGM/HvAwsQyAm2BRDBnKzA8AMzDNAHJgZhnTmAHhg8MtyHoksPp1h/hhJpKeZaWKK9zFyZoLHw/ORAinEHPK3hUwOm1qvC5XiuSFJPuWcUyJC/o045RMkCCQyOaUCiQABKpWaIBITEMgUgBCgAggItxsNwbnANzkpBueIga3QvmOoAAszE6CLas6lhAjAAsPw/m6wDBHW1xghwsIQyMcMQIAY4zd5uR9cXBGGFBmW+BI/srzM02//Di/Dx/vlN+Nn5KVwnMcJhrRQOE8wTfMC6WmcgPs/Pew7V26MiQCw2tRXZwoom1dJipk3juaVhRN5jW2u37uCw1WFSAQIV35n9XFyyBqyIgpKwMGVJ0O0znQ8giC46q5hIVOfgWU5ADGIwhxWDuP8AhAzbTmcByKBBWCAsJADcWPICBNAIDu2Hp0IDAucx/nlZeLMeL8s54/LZziPI/iyAP7jspSyfEOVaSGRPsF5hF8fDj3H8A2QsOJByCwDC8PRFwAfbkiSWKNxOVxT+XmLH8MqBEAUqgOuRj14KFewKzmQydfPagDAbUiKoTtsQGxTwfVb82UCOI/ncZyHcVhQIROF5TAsR7hfCBQ4Hr6MkY3isDTi9HDO5KvsCqRIAHCwwMRKE59/M8AH5sf0Mk0lwVDK8hGGAsA8kc5+5Erj0wsUpnGePx6O/8c/7vd5NSy5gZOBjE2FUiBFgMyKYyJFAhngGEmwCddwWjGtrjoUIokClMwV7woBRMiiXMlVFwjsIyGMah2OiIGGUJ25cmU/fjMrYZon7s9Pi0DIKFMA8pcI74+LsgQ4wwI5j8BxgRmCJy3kQIgp88oABAtpBfH65IHxnoXIzGF5YvKkM0Tw6TxP83Q/MgzLGWAaoHB8YpqI8+fnQQJXARkhrrpkaDQfPf6GJIEE1FWXHEm/UUyJRF7tRfIqRKsmTUkTieg9rG4idVW6OShAAIcLBL+XEKZRvMcRCVawgOtmm4PITUHM6Y2gCeQExHgYvB9yyuSsQGKBgMJa6NcyMRwe109aIM0AN0voNxTPTDHxyAv3vMDBLzMMBwbmK34e0ny9pwAv55HzMM2/Js0QsskkSEDMQGawNDw+JhJwXfSNYoa02ko5317LVKAqAJVESpScyTkDtSoggIYcQAI4CEw+yhgO3uKNYyQYAMFhrkD2zvAA48D5PAOrEhFKBjJpGHh/CGSAtKC6cAYiKC1RAKXllCHAQjgcLMDGlQs8DsA4xJnIPMF5jJQFpgi8MDNzP8RbJAkG7z3MjPzjgS//y2kDZbtrw9KGATzEq7KJwKrvVluTFdCUto4bTeAgpRvcZc5kQgCqI6KAIhlAHA7C1aULEXAIDo8VrAFYB64kAFgWWICmXClBBDILQ5wONwdxAMjAyAAAK6gADDBnJgdxSyIynD9y//LER/7464/Dy/3HX47zOHjSmZHEUj59hI/jAgvLCPN4f0tTTsw/8ddfh4e9SSHHBFehTA6LXbBL8YUN5pggxW8PKHgSiRQT8dsKCZcixMTtqdy00SolnUJFAOGWPKouQPBRIrsoajEdJzQBAIfZxnkOwFkWHhNLADQzcqdyPEBg+YknPCGTj4cQrmwJ54CokGEEUZV8GM+HJxXJYAcGpmn/bwTDAExTTPPMwMSwzPf+5TzdD8t8q2l7YQKGBEtiYpqmeKu/hPF+4vn5lDAkIEAgE5ahDTRiATaKK5IRPAUohQRXiFNKbIoJEoU3SOIQUEAByQEHDghEibKLgje+OweIcGNK+lo/f6XItjeWfIaVzk8zAI0lQxxAAEYACIwQECGMjASBkMr/BYXif+JpLNp7AAAAAElFTkSuQmCC",
```

Comment:
Hi @thiagofogaca25,

your logo uses an individual's image, especially a well-known personality. Without explicit permission can lead to legal issues related to copyright and image rights.
Please, update the PR.

Thanks!